pkg/utils/k8sutil.go

```diff
@@ -59,7 +60,9 @@ const (
 	node8Http      = "bitnami/kubeless-nodejs@sha256:1eff2beae6fcc40577ada75624c3e4d3840a854588526cd8616d66f4e889dfe6"
 	node8Pubsub    = "bitnami/kubeless-nodejs-event-consumer@sha256:4d005c9c0b462750d9ab7f1305897e7a01143fe869d3b722ed3330560f9c7fb5"
 	ruby24Http     = "bitnami/kubeless-ruby@sha256:9ac4a8a05f8f6ba023c1262943cca48efad0f271af884f306814db8ab0adeca5"
+	busybox        = "tuna/busybox@sha256:3651f7ee3b1e779471e338dcc43e6a5e69e0a8c7a4d08fd5702531cbbacc3269"
```

Comment:
this should not be from `tuna/` - was there something you needed that wasn't provided by one of the "official" busybox images?

Reply:
It's `curl`. I found the official busybox doesn't provide curl so I embedded it and packed a new one.

Reply:
Consider using `wget` or `nc`, which _are_ in the default `busybox` container.  My point here though is that if kubeless needs a custom container, then it should be built and managed as a part of the rest of kubeless - not a personal side project.

Reply:
Fixed. I turn it to `busybox` and executed `wget -qO-`